.release-please-manifest.json

```diff
@@ -3,9 +3,9 @@
   "circom-mpc-vm": "0.2.0",
   "circom-snarks": "0.0.0",
   "circom-types": "0.3.0",
-  "collaborative-circom": "0.2.1",
-  "collaborative-groth16": "0.2.1",
-  "collaborative-plonk": "0.0.0",
+  "co-circom": "0.2.1",
+  "co-groth16": "0.2.1",
+  "co-plonk": "0.0.0",
```

Comment:
i don't think we should put 222d147f7a9b5afc71b314d6eb9b4fa423a4c9ff in this PR, and rather sync on it later on in a separate PR tomorrow

Reply:
Removed it from the PR